src/Notes/WC_Admin_Notes_Start_Dropshipping_Business.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * WooCommerce Admin: Starting a dropshipping business.
+ *
+ * Adds a note to ask the client if they are considering starting a dropshipping business.
+ *
+ * @package WooCommerce Admin
+ */
+
+namespace Automattic\WooCommerce\Admin\Notes;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC_Admin_Notes_Start_Dropshipping_Business.
+ */
+class WC_Admin_Notes_Start_Dropshipping_Business {
+	/**
+	 * Note traits.
+	 */
+	use NoteTraits;
+
+	/**
+	 * Name of the note for use in the database.
+	 */
+	const NOTE_NAME = 'wc-admin-start-dropshipping-business';
+
+	/**
+	 * Get the note.
+	 */
+	public static function get_note() {
+
+		// We want to show the note after one day.
+		if ( ! self::wc_admin_active_for( DAY_IN_SECONDS ) ) {
+			return;
+		}
+
+		$onboarding_profile = get_option( 'woocommerce_onboarding_profile', array() );
+		if (
+			! isset( $onboarding_profile['setup_client'] ) ||
+			! isset( $onboarding_profile['product_count'] ) ||
+			! isset( $onboarding_profile['revenue'] )
```

Comment:
This causes the same logic issues as https://github.com/woocommerce/woocommerce-admin/pull/4441/files#r432634412

Reply:
Thank you @jeffstieler, you are right. It was corrected in the commit 9128c52